src/neuroconv/datainterfaces/ecephys/spikeglx/spikeglxdatainterface.py

```diff
@@ -54,38 +53,58 @@ def _source_data_to_extractor_kwargs(self, source_data: dict) -> dict:
     @validate_call
     def __init__(
         self,
-        file_path: FilePath,
-        verbose: bool = True,
+        folder_path: Optional[DirectoryPath] = None,
+        stream_id: Optional[str] = None,
         es_key: Optional[str] = None,
+        verbose: bool = True,
+        file_path: Optional[FilePath] = None,
```

Comment:
Let's preserve the order of the args so we do not cause a breaking change for ordered args